### PR TITLE
Ux/ui button redesign to increase click through

### PR DESCRIPTION
**Pull request by nweat**
changes made to event info slide out:
- move and resize "more" button
- pull image from gravatar based on user profile name and use as a profile pic

### CODE REVIEW

**Comment by AndyatFocallocal** [comment]
@nweat exciting! Can you copy paste these changes into the fl-sleeper branch too please

**Comment by AndyatFocallocal** [comment]
@nweat I see an error has creeped in to the live site. I'm on my mobile right now so I can't check console, but you should see that the main page and map are zoomed out now, and clicking an icon on the map leads to a blank page

**Comment by AndyatFocallocal** [comment]
@nweat A little more research on the issue. It only occurs on a smaller screen so is presumably related to responsiveness. It does work ok on my desktop device using inspect element to make it show as a mobile device, it's just my phone it's not working correctly on (5.5inch screen, android, chrome).

there is also a possibility that one of Arty's PR's caused it yesterday as i'm not sure i checked them fully on my phone.

**Comment by vermiculite** [comment]
@AndyatFocallocal I tested on chrome on my desktop and see the error. My monitor is HD. 

1. 94eca18d747f2b90268a8170f5a4f4a8b85e286d.js?meteor_js_resource=true:151 TypeError: Cannot read property 'profile' of null
2. Cannot read property 'profile' of null
3. Cannot read property 'panTo' of null

So the first two look like maybe checking the logged in user but no user logged in. I am not logged in when checking you.

The third looks like the maps are expecting a result but no arrives and gives the error.

**Comment by AndyatFocallocal** [comment]
@vermiculite do you think those issues came from this commit? they sound quite different to the changes made here

File: imports/client/ui/pages/Map/EventsList/EventInfo/index.js [hunk 1]
```diff
@@ -79,6 +80,7 @@ class EventInfo extends Component {
 
     const categories = formatUtils.formatCategories(event.categories)
     const distance = formatUtils.formatMilesFromLocation(userLocation, event.address.location.coordinates)
+    const gravatar = Gravatar.getGravatar(user.profile.name, 80)
```
Comment (vermiculite) [inline imports/client/ui/pages/Map/EventsList/EventInfo/index.js]: If user is undefined there will be an error here.

**Comment by nweat** [comment]
@AndyatFocallocal I think it could be from my commit because I am using user.profile.name to get the email and if a user is not logged in that user would not be set and now I know that the user pics hould actually be that of the event creator which I would need to fix

**Comment by AndyatFocallocal** [comment]
@nweat ok, cool. i'll revert and then test to be sure :+1:

**Pull request merged**